rover_msgs/Cameras.lcm

```diff
@@ -1,6 +1,8 @@
 package rover_msgs;
 
 struct Cameras {
-    int8_t port_0;
-    int8_t port_1;
+    int32_t port_0;
+    int32_t port_1;
+    int32_t port_2;
+    int32_t port_3;
```

Comment:
Do we wanna just change it to int32_t ports[4]

Reply:
If you'd like, would be an easy change for us